lib/grid.rb

```diff
@@ -37,17 +29,15 @@ def self.row(txt)
   def setup_with_fleet
     if @fleet
       @fleet.each do |ship|
-        ship.location.each do |coordinates|
-          @matrix[coordinates[0]][coordinates[1]] = HIT_CHAR
-        end
+        ship.location.each {|coordinates| @matrix[coordinates.first][coordinates[1]] = HIT_CHAR}
```

Comment:
I care about Ruby style. Please use space before and after `{` Take a look at: https://github.com/bbatsov/ruby-style-guide#spaces-operators . But I would keep a previous version as more readable.